### DISCUSSION TITLE
river mod(osu!standard)

### DISCUSSION THREAD

**Discussion by iSimpFor** [ppy/osu#16915] [Ideas]
how the river mod works is that all notes will move in a random direction or direction you specified, with wraparound occurring at the ranked limits(this includes the cursor so big and long sliders will still be playable)

you can customize:
-the option to use custom movement directions
- -the direction of the notes
- -the speed at which all notes move at

you will have to modify your aim to account of the shift in note positions

**Comment by mk56-spn** [comment]
the problem with this, specifically the wrap around, is that it would make the maps essentially unplayable on a tablet ( due to limited space and absolute positioning) or borderline playable on mouse, as you are forced to  drift off your table. so not really feasible afaik
